### ISSUE TITLE
Environment Variables are not expanded

### ISSUE THREAD

**Issue by dariomt** [Caphyon/clang-power-tools#207]
Some files in my projects are referenced though environment variables.

However, clang-power-tools does not seem to expand those environment variables to their actual values.

E.g. if I have the following in my .vcxproj

`<ClCompile Include="$(THIRDPARTYLIBS)\TinyODBC\tiodbc.cpp" />`

I get the following error:

```
PROJECT: D:\SVNProjects\MyProj.vcxproj
Join-Path : Cannot find path 'D:\SVNProjects\$(THIRDPARTYLIBS)\TinyODBC\tiodbc.cpp' because it does not exist.
```

Also, the value of this variable is an *absolute* path, and it seems clang-power-tools is interpreting it as a *relative* path?

TLDR Visual Studio 2015:
1) expands the actual value of the environment variable when referencing a file and 
2) interprets it correctly as an absolute path

It would be great if clang-power-tools could replicate this behavior.

**Comment by hero101111** [comment]
Hello,

We've just pushed a fix that enables variable usage in `Clinclude` and `ClCompile' project items.
Thank you for your input.

Regards,
Gabriel

---

Please note that this fix is currently live only for the github repository.
The latest Visual Studio extension does not contain the fix. It will be included in the next release.

Until then, as a workaround, you can manually replace `clang-build.ps1` in the VS extension directory with the latest from github. To locate the extension directory search for `clang-build.ps1` in `%LOCALAPPDATA%`

Don't forget to restart Visual Studio after replacing the script file.
